Physicist/Physicist/Controls/SoundController.cs

```diff
@@ -0,0 +1,191 @@
+            return id;
+        }
+
+        public static Guid PlayLocalizedSound(string name, AudioEmitter emitter)
+        {
+            SoundEffect effect = ContentController.Instance.GetContent<SoundEffect>(name);
+            SoundEffectInstance instance = effect.CreateInstance();
+
+            if (emitter != null)
+            {
+                instance.Apply3D(SoundController.listener, emitter);
+            }
+
+            if (SoundController.instances == null)
+            {
+                SoundController.instances = new Dictionary<Guid, SoundEffectInstance>();
+            }
+
+            Guid id = new Guid();
+            SoundController.instances.Add(id, instance);
+            instance.Play();
+            return id;
+        }
+
+        public static Guid PlayLocalizedSound(string name, AudioEmitter emitter, bool looping, float volume, float pitch, float pan)
+        {
+            SoundEffect effect = ContentController.Instance.GetContent<SoundEffect>(name);
+            SoundEffectInstance instance = effect.CreateInstance();
+            
+            instance.Volume = volume;
+            instance.Pitch = pitch;
+            instance.Pan = pan;
+            instance.IsLooped = looping;
+
+            if (emitter != null)
+            {
+                instance.Apply3D(SoundController.listener, emitter);
+            }
+
+            if (SoundController.instances == null)
+            {
+                SoundController.instances = new Dictionary<Guid, SoundEffectInstance>();
+            }
+
+            Guid id = new Guid();
+            SoundController.instances.Add(id, instance);
+            instance.Play();
+            return id;
+        }
+
+        public static void StopSound(Guid id)
+        {
+            if (SoundController.instances.ContainsKey(id))
+            {
+                SoundEffectInstance instance = SoundController.instances[id];
+                if (instance.State == SoundState.Playing)
+                {
+                    instance.Stop();
+                    instance.Dispose();
+                    SoundController.instances.Remove(id);
+                }
+            }
+        }
+
+        public static void PauseSound(Guid id)
+        {
+            if (SoundController.instances.ContainsKey(id))
+            {
+                SoundEffectInstance instance = SoundController.instances[id];
+                if (instance.State == SoundState.Playing)
+                {
+                    instance.Pause();
+                }
+            }
+        }
+
+        public static void ResumeSound(Guid id)
+        {
+            if (SoundController.instances.ContainsKey(id))
+            {
+                SoundEffectInstance instance = SoundController.instances[id];
+                if (instance.State == SoundState.Paused)
+                {
+                    instance.Resume();
+                }
+            }
+        }
+
+        public static bool SoundIsPlaying(Guid id)
```

Comment:
The Comment is for SoundIsPlaying, strive to make methods with only one return statement